xbmc/FileItem.cpp

```diff
@@ -490,8 +476,7 @@ void CFileItem::Reset()
   m_epgInfoTag.reset();
   delete m_pvrChannelInfoTag;
   m_pvrChannelInfoTag=NULL;
-  delete m_pvrRecordingInfoTag;
-  m_pvrRecordingInfoTag=NULL;
+  m_pvrRecordingInfoTag.reset();
```

Comment:
we better assign a new empty one here, in case one got copied, is used elsewhere, but is reset here.